### PR TITLE
feat: mvp flow

### LINKED ISSUE #63: [Bug]: mvp flow
### Expected Behavior

_No response_

### Actual Behavior

- [x]  fix incorrect email address in checkout customer step
- [ ] add filter functionality
- [x] use order.code instead of order.id
- [x] finish product variant functionality
- [x] develop isInCart
- [x] finish apply coupon functionality
- [x] move metadata back to theme in setPaymentMethod
- [x] add info to docs about local development (bearer token and disable register email verification in Vendure server)

### Possible Solution

_No response_

### Steps to reproduce

_No response_

### What version of vendure integration are you using?

0.1.0

### What version of Node.js are you using?

14.0.0

### What browser (and version) are you using?

chrome

### What operating system (and version) are you using?

macOS

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Baroshem**
## Description

Closes #63 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/composables/__tests__/useCart/useCart.spec.ts [hunk 1]
```diff
@@ -99,9 +100,16 @@ describe('[vendure-composables] useCart', () => {
...
       customQuery
     );
   });
+
+  it('checks if product is in cart', async () => {
+    const { isInCart } = useCart() as any;
+    const response = await isInCart(context, { currentCart: mockedCart, product: { _id: '4' } });
+
+    expect(response).toEqual(true);
+  });
```
Comment (Mateuszpietrusinski) [inline packages/composables/__tests__/useCart/useCart.spec.ts:114]: Please provide another test case where you will be checking is product in the cart when the cart is empty

File: packages/theme/pages/Category.vue [hunk 2]
```diff
@@ -164,7 +164,7 @@
               :score-rating="productGetters.getAverageRating(product)"
               :show-add-to-cart-button="true"
               :isOnWishlist="isInWishlist({ product })"
-              :isAddedToCart="isInCart({ product })"
+              :isAddedToCart="isInCart({ cart, product })"
```
Comment (Mateuszpietrusinski) [inline packages/theme/pages/Category.vue:167]: AFAIK the cart is available in isInCart method. The factory function implementation:

```
const isInCart = ({ product }) => {
      return _factoryParams.isInCart({
        currentCart: cart.value,
        product
      });
    };
```

So, you don't need to pass cart as a param

File: packages/theme/pages/Checkout/Customer.vue [hunk 3]
```diff
@@ -128,15 +131,20 @@ export default {
     });
 
     const handleFormSubmit = async () => {
-      await $vendure.api.setCustomerForOrder(form.value);
+      const response = await $vendure.api.setCustomerForOrder(form.value);
```
Comment (Mateuszpietrusinski) [inline packages/theme/pages/Checkout/Customer.vue:134]: Maybe a better option for sending a form is to make a composable, instead of directly calling api-client function

**Review by Mateuszpietrusinski — COMMENTED** [review]
Three comments

File: packages/theme/pages/Checkout/Customer.vue (hunk 3 above)
Comment (Baroshem) [reply packages/theme/pages/Checkout/Customer.vue:134]: I will create an issue for that. This would require a new composable and factory and I will develop it for stable 1.0.0 version. For now we would release this MVP as 1.0.0-rc.1
